background.js

```diff
@@ -1,68 +1,59 @@
-let exclusionArrayOption = [];
-let moveTabsOption;
-let effectWindowsOption;
-let effectTabGroupsOption;
+let options = {};
+const defaultOptions = {
+    moveTabs: true,
+    effectWindows: false,
+    effectTabGroups: false,
+    exclusions: [],
```

Comment:
Try not to name variables after what type they are. 
`exclusionArray` -> `exclusions`. We know its an array by looking at the `[]`.